stanford_profile_helper.module

```diff
@@ -609,11 +610,46 @@ function stanford_profile_helper_preprocess_pattern_localfooter(&$variables) {
  * Implements hook_form_FORM_ID_alter().
  */
 function stanford_profile_helper_form_media_library_add_form_embeddable_alter(array &$form, FormStateInterface $form_state) {
+
+  $source_field = $form_state->get('source_field');
+  $embed_code_field = $form_state->get('unstructured_field_name');
   $user = \Drupal::currentUser();
   $authorized = $user->hasPermission('create field_media_embeddable_code')
     || $user->hasPermission('edit field_media_embeddable_code');
 
-  if (isset($form['container']['field_media_embeddable_code'])) {
-    $form['container']['field_media_embeddable_code']['#access'] = $authorized;
+  if (isset($form['container'][$embed_code_field])) {
+    $form['container'][$embed_code_field]['#access'] = $authorized;
+  }
+
+  if (isset($form['container'][$source_field])) {
+    $new_desc = 'Allowed providers: @providers. For custom embeds, please <a href="@snow_form">request support.</a>';
+    $args = $form['container'][$source_field]['#description']->getArguments();
+    $args['@snow_form'] = 'https://stanford.service-now.com/it_services?id=sc_cat_item&sys_id=83daed294f4143009a9a97411310c70a';
+    $form['container'][$source_field]['#description'] = new FormattableMarkup($new_desc, $args);
```

Comment:
does `t()` function not work here?

Reply:
t() got me a TranslatableMarkup object instead of a FormattableMarkup object. I suspect there is little difference functionally in this case.

Reply:
they result in the same thing. lets use the conventional `t()` function instead.

Reply:
Fixed!